docs/data/date-pickers/custom-field/BrowserV7Field.js

```diff
@@ -44,6 +44,9 @@ const BrowserTextField = React.forwardRef((props, ref) => {
     // Can be passed to a hidden <input /> element
     onChange,
     value,
+    // Can be passed to the button that clears the value
+    clearable,
+    onClear,
```

Comment:
this triggered me a bit ... since above it is "handler", "value" and here it is reversed

Clearly a massive nitpick and fine as is!!! :P

Reply:
:laughing: Nitpick fixed